services/autoscaling/src/simcore_service_autoscaling/modules/auto_scaling_task.py

```diff
@@ -28,6 +29,13 @@ async def _startup() -> None:
                     "node_labels": app_settings.AUTOSCALING_NODES_MONITORING.NODES_MONITORING_NODE_LABELS
                 }
             )
+        elif app_settings.AUTOSCALING_DASK:
+            lock_key += (
```

Comment:
MINOR: I think it would be less error-prone (specially with the separators `:`) to simply have a list of "labels" and just join them when you pass to the `start_periodic_task`, e.g.
```python
lock_key_parts = [app.title, app.version]
...
lock_key_parts += ["dynamic", app_settings.AUTOSCALING_NODES_MONITORING.NODES_MONITORING_NODE_LABELS ]
...


start_periodic_task(..., lock_key =":".join(f"{k}" for k in lock_key_parts), ...)
```

Reply:
agree I like your example